app/forms/work_packages/progress_form.rb

```diff
@@ -199,9 +197,10 @@ def as_percent(value)
   end
 
   def default_field_options(name)
+    action = name == :status_id ? "change" : "input"
     data = { "work-packages--progress--preview-progress-target": "progressInput",
              "work-packages--progress--touched-field-marker-target": "progressInput",
-             action: "input->work-packages--progress--touched-field-marker#markFieldAsTouched" }
+             action: "#{action}->work-packages--progress--touched-field-marker#markFieldAsTouched" }
```

Comment:
Now that I think of it, we could even omit the action and it would use the default event for each input type. Meaning for select fields it would use "change", for input fields would use "input" and we don't need to specify this. But we could also argue that being explicit about the event is probably best.
```suggestion
             action: "work-packages--progress--touched-field-marker#markFieldAsTouched" }
```

Reply:
Good catch! I tried it and it works. I'll commit it.

Reply:
commited